spec/models/school_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'rails_helper'
+
+RSpec.describe School, type: :model do
```

Comment:
I wonder whether the time is right to introduce https://github.com/thoughtbot/shoulda-matchers, which would make this spec much more concise...

Reply:
As discussed, defer until later.